src-docs/spec/sendEphemeralUpdate.md

```diff
@@ -0,0 +1,9 @@
+# sendEphemeralUpdate
+
+```js
+window.webxdc.sendUpdate(payload);
+```
+
+Send an ephemeral update to all peers. [setEphemeralUpdateListener](./setEphemeralUpdateListener.md) has to be called first in order to join the gossip swarm and make it operational. Only after the promisle from [setEphemeralUpdateListener](./setEphemeralUpdateListener.md) resolves, the sent updates are able to reache a peer.
```

Comment:
what "gossip swarm"?   Please keep such iroh implementation terms out of the spec. XMPP folks might have a completelly different method for transmitting ephemeral updates.